README.md

```diff
@@ -1 +1,60 @@
-# To-Do-list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+To Do list
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo (if available)
+
+[Live Demo Link]
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
```

Comment:
- [ ] I kindly suggest that you add the prerequisites or remove this section entirely because it doesn't make your code neat.